middleware/localNamingConventions.js

```diff
@@ -0,0 +1,39 @@
+
+function setup() {
+  return applyLocalNamingConventions;
+}
+
+function applyLocalNamingConventions(req, res, next) {
+
+  // do nothing if no result data set
+  if (!res || !res.data) {
+    return next();
+  }
+
+  // loop through data items and flip relevant number/street
+  res.data.filter(function(place){
+    // only relevant for German addresses
+    if( 'DEU' !== place.alpha3 ){ return false; }
```

Comment:
Yup, maybe the next time someone adds a country, or maybe even the time after that, this will need some sort of config. Until then this code nicely fixes an issue for our friends in Germany :)